parsec/core/cli/import_device.py

```diff
@@ -0,0 +1,70 @@
+# Parsec Cloud (https://parsec.cloud) Copyright (c) AGPLv3 2016-2021 Scille SAS
+from pathlib import Path
+
+import click
+
+from parsec.cli_utils import cli_exception_handler, spinner
+from parsec.core import CoreConfig
+from parsec.core.cli.utils import (
+    cli_command_base_options,
+    core_config_options,
+    format_available_devices,
+)
+from parsec.core.local_device import load_device_with_password
+from parsec.core.recovery import create_new_device_from_original
+from parsec.core.types import LocalDevice
+from parsec.utils import trio_run
+
+
+async def _import_device(
+    config: CoreConfig,
+    destination: Path,
+    recovery_password: str,
+    new_password: str,
+    device_label: str,
+) -> None:
+
+    async with spinner("Loading recovery device with password"):
+        recovery_device: LocalDevice = load_device_with_password(
+            destination, password=recovery_password
+        )
+
+    async with spinner("Creating new device from recovery"):
+        new_device = await create_new_device_from_original(
+            recovery_device, device_label, new_password, config.config_dir
+        )
+    device_display = click.style(new_device.slughash, fg="yellow")
+    click.echo(f"Saving device {device_display}")
+
+    click.echo("New device imported:")
+    click.echo(format_available_devices([new_device]))
+
+
+# .psrk stands for ps(parsec)r(recovery)k(key)
+@click.command(short_help="import device")
+@click.option("--key-path", "-K", help="Path of the device key to recover (.psrk)", required=True)
+@click.option("--recovery_password", "-R", help="Password of the recovery device", required=True)
```

Comment:
You should use click's `@click.password_option` here (see what is done in `parsec/core/cli/invitation.py`)